pallets/parachain-staking/src/migrations.rs

```diff
@@ -16,184 +16,34 @@
 
 //! # Migrations
 
-use crate::{types::RoundInfo, Config, RoundIndex};
-use frame_support::pallet_prelude::*;
-use frame_support::storage::generator::StorageValue;
-use frame_support::storage::unhashed;
-use frame_support::traits::OnRuntimeUpgrade;
-use frame_system::pallet_prelude::*;
-use sp_runtime::Saturating;
+//use crate::{types::RoundInfo, Config, RoundIndex};
+//use frame_support::pallet_prelude::*;
+//use frame_support::traits::OnRuntimeUpgrade;
+//use frame_system::pallet_prelude::*;
```

Comment:
```suggestion
```

Reply:
Deleted in https://github.com/moonbeam-foundation/moonbeam/pull/2885/commits/a9f0e756af1ad0a0170f187a09b0a6cd1d059a63